sys/src/9/port/port.json

```diff
@@ -21,6 +21,7 @@
 			"../port/devcoreboot.c",
 			"../port/devdup.c",
 			"../port/devenv.c",
+			"../port/devether.c",
```

Comment:
if you put this in, e.g., amd64/whatever.json, we would not need these fake ethernet devices in riscv/. That seems better to me.